org.adempiere.base.process/src/org/compiere/process/OrderLineCreateProduction.java

```diff
@@ -79,22 +79,21 @@ protected String doIt () throws Exception
 	{
 		if (log.isLoggable(Level.INFO)) log.info("C_OrderLine_ID=" + p_C_OrderLine_ID );
 		if (p_C_OrderLine_ID == 0)
-			throw new IllegalArgumentException("No OrderLine");
+			throw new IllegalArgumentException(Msg.getMsg(Env.getCtx(), "No OrderLine"));
 		//
 		MOrderLine line = new MOrderLine (getCtx(), p_C_OrderLine_ID, get_TrxName());
 		if (line.get_ID() == 0)
-			throw new IllegalArgumentException("Order line not found");
+			throw new IllegalArgumentException(Msg.getMsg(Env.getCtx(), "Order line not found"));			
 		MOrder order = new MOrder (getCtx(), line.getC_Order_ID(), get_TrxName());
 		if (!MOrder.DOCSTATUS_Completed.equals(order.getDocStatus()))
-			throw new IllegalArgumentException("Order not completed");
-		
+		    throw new IllegalArgumentException(Msg.getMsg(Env.getCtx(), "Order not completed"));
 		MDocType doc = new MDocType(getCtx(), order.getC_DocType_ID(), get_TrxName());
 		
 		if ( (line.getQtyOrdered().subtract(line.getQtyDelivered())).compareTo(Env.ZERO) <= 0 )
 		{
 			if (!doc.getDocSubTypeSO().equals("ON"))  //Consignment and stock orders both have subtype of ON
-			{
-			    return "Ordered quantity already shipped";
+			{			    
+			    return Msg.getMsg(Env.getCtx(), "Ordered quantity already shipped");

```

Comment:
It seems there's no message record created for the 3 Msg.getMsg above ? Overlook ?

Reply:
The other messages were previously created with ticket IDEMPIERE-4035  and I'm reusing them.